c_src/device/bcm.c

```diff
@@ -251,33 +253,33 @@ void device_poll()
 
 void device_begin_render(driver_data_t* p_data)
 {
-  NVGcontext* p_ctx = p_data->p_ctx;
+  NVGcontext* v_ctx = p_data->v_ctx;
```

Comment:
```suggestion
  NVGcontext* p_ctx = (NVGcontext*)p_data->v_ctx;
```
Then the rest of the function doesn't need to change

Reply:
Makes sense (re https://github.com/ScenicFramework/scenic_driver_local/issues/48#issuecomment-1705606724). Updated in 37e1b7d982074af512ae06036103094a56157b6f